### ISSUE TITLE
grpc_proto.bzl out of date?

### ISSUE THREAD

**Issue by pcj** [mzhaom/trunk#16]
I cloned the repo/submodules and tried to build the examples.  It fails quickly with the following error.  I am building with bazel 0.3.1rc2, but the same error occurs with 0.3.0.  I have not tried earlier bazel versions. 

```
bazel build examples/...:all
trunk/third_party/grpc/grpc_proto.bzl:116:12: Attribute 'attrs' either doesn't exist or uses a select() (i.e. could have multiple values)
Available attributes: $protoc, :action_listener, compatible_with, deprecation, deps, expect_failure, features, gen_cc, gen_java, generator_function, generator_location, generator_name, grpc_cpp_plugin, grpc_java_plugin, has_service, restricted_to, src, tags, testonly, visibility
```

**Comment by pcj** [comment]
Also see https://groups.google.com/d/msg/bazel-discuss/Crs4RNqHoCI/7Otj-W-6BgAJ

**Comment by bowlofstew** [comment]
I was able to repro this locally as well.

**Comment by michaeljs1990** [comment]
Yeah i have been running into a slew of different issues building this with 0.3.1. However the examples are great and pretty easy to fix once you take a look over things. Not sure if this project aims to support 0.3.1 it looks like it's stuck at testing against 0.2.2 right now.

**Comment by mzhaom** [comment]
Yeah, I realize it's kind of out of date. We are slowly picking up 0.3.1 internally. So hopefully I will take a chance to fix this issue in a few days.

**Comment by michaeljs1990** [comment]
@mzhaom your boost stuff helped me out significantly. I have added a few more libraries in my repo here https://github.com/michaeljs1990/clog/tree/master/third_party/boost if of any use to you.

**Comment by bowlofstew** [comment]
@michaeljs1990 likewise and nice work! I'm going to pull that over into my fork.
